vspace/src/main/java/edu/asu/diging/vspace/core/services/impl/ExhibitionManager.java

```diff
@@ -66,4 +67,13 @@ public IExhibition getStartExhibition() {
         }
         return null;
     }
+    
+    @Override
+    public IExhibition updateSpaceOrderMode(ExhibitionSpaceOrderMode mode) {
+        // for now we just take the first one created, there shouldn't be more than one
+        List<Exhibition> exhibitions = exhibitRepo.findAllByOrderByIdAsc();
+        Exhibition exhibition = exhibitions.get(0);
```

Comment:
just call `getStartExhibition`? this will take care of any checks etc.

Reply:
Done